pymc_experimental/inference/pathfinder.py

```diff
@@ -0,0 +1,96 @@
+import collections
+import sys
+
+import arviz as az
+import blackjax
+import jax
+import jax.numpy as jnp
+import jax.random as random
+import numpy as np
+import pymc as pm
+from pymc import modelcontext
+from pymc.sampling_jax import get_jaxified_graph
+from pymc.util import get_default_varnames
+
+
+def convert_flat_trace_to_idata(
+    samples,
+    dims=None,
+    coords=None,
+    include_transformed=False,
+    postprocessing_backend="cpu",
+    model=None,
+):
+    model = modelcontext(model)
+    init_position_dict = model.initial_point()
+    trace = collections.defaultdict(list)
+    astart = pm.blocking.DictToArrayBijection.map(init_position_dict)
+    for sample in samples:
+        raveld_vars = pm.blocking.RaveledVars(sample, astart.point_map_info)
+        point = pm.blocking.DictToArrayBijection.rmap(raveld_vars, init_position_dict)
+        for p, v in point.items():
+            trace[p].append(v.tolist())
+
+    print("Creating trace...", file=sys.stdout)
```

Comment:
I would remove this

```suggestion
```

Reply:
I would also find a better place for that, the utility seems to be useful